lib/core/viewer/components/viewer.component.scss

```diff
@@ -4,6 +4,10 @@
     background-color: var(--theme-card-bg-color);
 }
 
+.adf-datatable-container {
+    height: auto !important;
```

Comment:
why the viewer should know about the datatable?

Reply:
I have updated the changes